pkg/security/ptracer/cws.go

```diff
@@ -73,7 +74,7 @@ type syscallID struct {
 }
 
 type syscallHandler struct {
-	IDs        []syscallID        // IDs defines the list of syscall IDs related to this handler
+	ID         syscallID          // ID identifies the syscall related to this handler
```

Comment:
What's the point of changing the data structure here? It implies more code repetitions and I don't get the benefits

Reply:
The benefit is that we can directly access the syscall ID from the corresponding handler like so:
https://github.com/DataDog/datadog-agent/pull/27795/files#diff-bc302f5cd1ee2f6951b5cdbee920ef217a5bb353c878dca6fb3748f8d4ad36baR487

Reply:
I see! Yep, it could be convenient for debug. Thanks!